test-requirements.in

```diff
@@ -13,8 +13,7 @@ cryptography>=41.0.0  # cryptography<41 segfaults on pypy3.10
 black; implementation_name == "cpython"
 mypy; implementation_name == "cpython"
 types-pyOpenSSL; implementation_name == "cpython" # and annotations
-flake8
-flake8-pyproject
+ruff >= 0.0.291
```

Comment:
Unless we're depending on some 0.0.291 functionality, I think we should not have a version constraint here -- the lockfile can have that for us!

Reply:
`--format` is depreciated, 0.0.291 has the new version, `--output-format` dcce938225bc22436ff3935b5e3f7d215a8f0b32